import-export-cli/build.sh

```diff
@@ -110,6 +113,7 @@ do
 
     cp -r "${baseDir}/resources/README.html" $zipdir > /dev/null 2>&1
     cp -r "${baseDir}/LICENSE" $zipdir > /dev/null 2>&1
+    cp -r "${baseDir}/mi/encryption-client/target/encryption-client-$build_version.jar" $zipdir > /dev/null 2>&1
```

Comment:
This PR introduces some Java code for the purpose of secret encryption which will be bundled into the distribution as a jar file to be executed by the ctl. This would make the ctl effectively Java dependent and it loses the advantage of being a standalone executable. 

We need to come up with a purely Golang implementation to meet this requirement. https://github.com/pavel-v-chernykh/keystore-go can be used to get key store functionality in Go if required.

Reply:
Implemented the functionality using Golang in https://github.com/wso2/product-apim-tooling/pull/581